packages/console-client/tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "target": "ES2022",
+    "lib": ["ES2022"],
+    "module": "ESNext",
```

Comment:
Now it is a ESM module, we should add it in the README (and probably also in the CHANGELOG)

Reply:
is the changelog in use? don't we use releases? Let's change the PR title as well to denote this change so that it is properly mapped in the release description

Reply:
done